server/chat-plugins/quotes.ts

```diff
@@ -37,13 +37,29 @@ export const commands: ChatCommands = {
 		room = this.requireRoom();
 		const roomQuotes = quotes[room.roomid];
 		if (!roomQuotes?.length) return this.errorReply(`This room has no quotes.`);
-		this.runBroadcast(true);
 		const {quote, date, userid} = roomQuotes[Math.floor(Math.random() * roomQuotes.length)];
 		const time = Chat.toTimestamp(new Date(date), {human: true});
 		const attribution = toID(target) === 'showauthor' ? `<hr /><small>Added by ${userid} on ${time}</small>` : '';
 		return this.sendReplyBox(`${Chat.getReadmoreBlock(quote)}${attribution}`);
 	},
 	randquotehelp: [`/randquote [showauthor] - Show a random quote from the room. Add 'showauthor' to see who added it and when.`],
+	
+	nquote(target, room, user) {
+		room = this.requireRoom();
+		const roomQuotes = quotes[room.roomid];
+		if (!roomQuotes) return this.errorReply(`No quotes found for ${room.roomid}.`);
+		const index = parseInt(target.trim());
+		if (isNaN(index)) {
+			return this.errorReply(`Invalid index.`);
+		}
+		if (!roomQuotes[index - 1]) {
+			return this.errorReply(`Out of bounds.`);
+		}
+		const {quote, date, userid} = roomQuotes[index - 1];
+		const time = Chat.toTimestamp(new Date(date), {human: true});
+		return this.sendReplyBox(`${Chat.getReadmoreBlock(quote)}`);
```

Comment:
```suggestion
		return this.sendReplyBox(Chat.getReadmoreBlock(quote));
```
Just cleaning this up a bit, no need to place this in a string if the result is the entire string.